tailwind.config.js

```diff
@@ -75,6 +75,7 @@ module.exports = {
           1: '#656565',
           2: '#8E8E8E',
           3: '#D9D9D9',
+          4: '#f5f8fa',
         },
 
```

Comment:
Huge spaces appear in between headlines and paragraphs:

![image](https://user-images.githubusercontent.com/88790097/135609462-8326d398-cde8-494c-82b8-f94d88d06ff7.png)

Reply:
Cannot reproduce this issue